sycl/gdb/libsycl.so-gdb.py

```diff
@@ -35,7 +35,7 @@ def index(self, arg):
             arg = arg["MImpl"]["MIndex"]
         except:
             pass
-        # https://github.com/intel/llvm/blob/97272b7ebd569bfa13811913a31e30f926559217/sycl/include/CL/sycl/accessor.hpp#L678-L690
+        # https://github.com/intel/llvm/blob/97272b7ebd569bfa13811913a31e30f926559217/sycl/include/sycl/accessor.hpp#L678-L690
```

Comment:
The link here and below is 404. Please update or remove.

Reply:
I'll remove once pre-commit testing finishes, thanks!